common/persistence/cassandra/execution_store.go

```diff
@@ -64,6 +64,10 @@ const (
 	rowTypeVisibilityTaskNamespaceID = "10000000-6000-f000-f000-000000000000"
 	rowTypeVisibilityTaskWorkflowID  = "20000000-6000-f000-f000-000000000000"
 	rowTypeVisibilityTaskRunID       = "30000000-6000-f000-f000-000000000000"
+	// Row constants for History task row.
+	rowTypeHistoryTaskNamespaceID = "10000000-7000-f000-f000-000000000000"
+	rowTypeHistoryTaskWorkflowID  = "20000000-7000-f000-f000-000000000000"
+	rowTypeHistoryTaskRunID       = "30000000-7000-f000-f000-000000000000"
 	// Row Constants for Replication Task DLQ Row. Source cluster name will be used as WorkflowID.
 	rowTypeDLQNamespaceID = "10000000-6000-f000-f000-000000000000"
 	rowTypeDLQRunID       = "30000000-6000-f000-f000-000000000000"
```

Comment:
hmm, notice that these 2 collide with rowTypeVisibilityTaskNamespaceID/rowTypeVisibilityTaskRunID

Reply:
😑 Looks like we'll have to live with it. Luckily we still have rowType to differentiate them.
Let me change the ID for history task row to be 8000 though, to be consistent with its rowType.